locator/token_metadata.cc

```diff
@@ -579,6 +579,7 @@ void token_metadata_impl::remove_endpoint(host_id endpoint) {
     _topology.remove_endpoint(endpoint);
```

Comment:
> Fixes https://github.com/scylladb/scylladb/issues/16730

The issue appears to be closed. Is this still happening?

Reply:
Nope